trunk/SUAVE/Plots/Geometry_Plots/plot_vehicle.py

```diff
@@ -39,286 +40,582 @@ def plot_vehicle(vehicle, save_figure = False, plot_control_points = True, save_
     N/A	
     """	
     # unpack vortex distribution 
-    VD = vehicle.vortex_distribution
-    
-    face_color = 'grey'        
-    edge_color = 'grey'
-    alpha_val  = 0.5  
-    
+    try:
+        VD = vehicle.vortex_distribution 
+    except:
+        settings = Data()
+        settings.number_spanwise_vortices  = 25
+        settings.number_chordwise_vortices = 5
+        VD = generate_wing_vortex_distribution(vehicle,settings)  
+        
     # initalize figure 
     fig = plt.figure(save_filename) 
-    fig.set_size_inches(8, 8) 
+    fig.set_size_inches(12, 12) 
```

Comment:
can you set this as a default? My computer can't display a 12x12 plot.

Reply:
Changed. Issue Resolved